beacon_node/network/src/beacon_processor/worker/rpc_methods.rs

```diff
@@ -246,19 +246,39 @@ impl<T: BeaconChainTypes> Worker<T> {
                                 "request_root" => ?root
                             );
                         }
-                        Ok((Some(_), None)) => {
-                            debug!(
-                                self.log,
-                                "Peer requested block and blob, but no blob found";
-                                "peer" => %peer_id,
-                                "request_root" => ?root
-                            );
-                            self.send_error_response(
-                                peer_id,
-                                RPCResponseErrorCode::ResourceUnavailable,
-                                "No blob for requested block".into(),
-                                request_id,
-                            );
+                        Ok((Some(block), None)) => {
+                            let data_availability_boundary_by_root = self.chain.data_availability_boundary_by_root_rpc_request();
+                            let block_epoch = block.epoch();
+
+                           if Some(block_epoch) >= data_availability_boundary_by_root {
+                                debug!(
```

Comment:
Nice! I really like having separate logs for these separate scenarios, cause it helps attribute the fault to us or the requestor